src/api_manager/context/request_context.cc

```diff
@@ -266,6 +266,32 @@ void RequestContext::FillLogMessage(service_control::ReportRequestInfo *info) {
   }
 }
 
+void RequestContext::FillHttpHeaders(service_control::ReportRequestInfo *info,
+                                     Response *response) {
```

Comment:
per c++ styple guide,  output should be the last one

so it should be

(const Response& response,  Info* info)